articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -14,19 +14,25 @@ Learn how to start writing quantum programs.
 
 ## Getting the basics: what do you need to know?
 
-Fortunately, you don’t need to know quantum physics to start writing quantum applications. However, there are some concepts that are especially important to understand quantum computing. But don’t worry if you don’t know something! We will give you the resources to learn the necessary tools.
+You don’t need to know quantum physics to learn about Q# and quantum computing or start writing quantum applications.
+
+To guide you in learning quantum computing with Q#, we provide a Getting Started Guide (link below) for the Quantum Development Kit, which provides a journey guide for both those who want to start coding quantum programs and those who are not yet ready to start coding, but want to learn more about Q# and quantum programming.
+
+Before embarking on the Getting Started journey, this last article on the “Five things to know about quantum computing” introduces you to some basic concepts that are especially important to understand quantum computing and points to further resources for you along your journey.
+
+These concepts will give you a good introduction to the fundamental knowledge you need to start coding quantum programs.  Those who are not yet ready to start coding, but want to learn more about Q# and quantum programming can still review these concepts at a high level here and then proceed to the Getting Started Guide to learn more about quantum computing and Q# without having to start coding.
 
 * [Linear algebra (vectors and matrices)](xref:microsoft.quantum.concepts.vectors): in quantum computing, quantum states are represented by vectors, with quantum operations being linear transformations applied to these vectors.
 * [Complex numbers](https://en.wikipedia.org/wiki/Complex_number): the coefficients of quantum state vectors are complex numbers. You can understand some basic quantum computing concepts without them, but you won't get far before you need to incorporate them into your quantum toolkit.
 * [Basic quantum mechanics](xref:microsoft.quantum.concepts.intro): We just said that you don’t need to know quantum physics to start coding (and it’s true!). But some basic concepts of quantum mechanics and its mathematical notation will be helpful to understand quantum programming.
 
-## Code first, then learn!
+## Code first, then learn! Or Learn more, then code!
 
-Now that you have the basics, you're ready to start writing your first quantum programs. There are many options to get started. Here we recommend you some of them:
+Now that you have the basics, you're ready to start writing your first quantum programs.  Or maybe you want to learn more before coding.  Either way, our Getting Started with the Quantum Development Kit provides a series of steps in learning, which includes:
```

Comment:
```suggestion
Now that you have the basics, you're ready to start writing your first quantum programs.  Or maybe you want to learn more before coding.  Either way, our [Getting Started with the Quantum Development Kit](URL??) provides a series of steps in learning, which includes:
```

Reply:
Seems like the Getting Started should have a single URL -- it sounds like a specific content thing.

Reply:
Fixed